.codecov.yml

```diff
@@ -0,0 +1,7 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 75
+        # Allow 0% coverage regression
+        threshold: 0
```

Comment:
This is nice actually :) 
I'm interested if we can enforce that without codecov, any idea ?

Reply:
I haven't got it to work with codecov. So may abandon this in the end. 

There is `pytest-cov` in tox with a hard percentage limit per tox environment run.

I set 75% as a low benchmark that we can ratchet up over time to kind of manually tighten the belt.

I'm curious to see the "coverage overtime" aspect of codecov but also happy to rip it out now or later. Just figured it'd be a good sign of project health if we add that badge like "we care".